docker-compose.yaml

```diff
@@ -204,9 +205,16 @@ services:
         build: ./tests
         networks:
             - default
+        environment:
+            - GEOSHOP_DEMO_LOGIN=admin:Test1234
+            - EXTRACT_DEMO_LOGIN=admin:motdepasse21
```

Comment:
it is better to get the  from the .env file if that is possible. It will be easier to maintain them ;-) 
On mid-term we will need to take a look ad the .env file any way so for me it is enough to just leave a TODO note here for the moment